Makefile

```diff
@@ -76,7 +76,7 @@ DOCKER_BUILDKIT=1
 export ACK_GINKGO_DEPRECATIONS := 1.16.4
 
 # Set --output-base for conversion-gen if we are not within GOPATH
-ifneq ($(abspath $(REPO_ROOT)),$(shell go env GOPATH)/src/sigs.k8s.io/cluster-api-provider-aws)
+ifneq ($(abspath $(REPO_ROOT)),$(abspath $(shell go env GOPATH)/src/sigs.k8s.io/cluster-api-provider-aws))
```

Comment:
Revert this given we have a different PR open?

Reply:
The change was necessary to generate bits in this PR, keeping a diff locally is not ideal so hopefully we just merge the other one today?

Reply:
We'll have to wait for feedback from the other maintainers in the other PR, we can revert this bit for now if we want to merge this one in and iterate on the rest of the codebase